bitcoin/src/blockdata/script/mod.rs

```diff
@@ -155,6 +159,46 @@ pub fn write_scriptint(out: &mut [u8; 8], n: i64) -> usize {
     len
 }
 
+/// Returns minimally encoded scriptint as a byte vector.
+pub fn scriptint_vec(n: i64) -> Vec<u8> {
+    let mut buf = [0u8; 8];
+    let len = write_scriptint(&mut buf, n);
+    buf[0..len].to_vec()
+}
```

Comment:
We already have `script::Builder::push_int`. Should look into the purpose of this method.